src/components/SearchField/SearchField.less

```diff
@@ -0,0 +1,29 @@
+.lucid-SearchField {
+
+	position: relative;
+
+	.lucid-TextField {
+
+		border-radius: @TextField-size-height / 2;
+		padding: 0 12px;
+
+		&:focus + .lucid-SearchIcon {
+			fill: @color-primary;
+		}
+
+	}
+
+	&-Icon {
+		position: absolute;
+		top: @TextField-size-height / 2 - 8;
```

Comment:
instead of positioning like this, you should wrap the icon with something like

```
<div style="
	position: absolute;
	right: 0;
	top: 0;
	height: 100%;
	display: flex;
	align-items: center;
	margin-right: 6px;
"></div>
```

so that if the icon, or text field height changes it will always be centered vertically